packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -90,6 +91,12 @@ export class GrpcPlugin extends BasePlugin<grpc> {
         'makeClientConstructor' as never,
         this._patchClient()
       );
+
+      shimmer.wrap(
+        this._moduleExports,
+        'makeGenericClientConstructor' as never,
```

Comment:
I'm curious, what's the role of `as never` here?

Reply:
Good point, it's not needed here. It was needed for other shimmer wraps because no types were provided for the internal `client.js` file, so I casted it `as any`. I've removed all of these `as never` by providing my own types for this internal module.

https://github.com/open-telemetry/opentelemetry-js/pull/325/files#diff-deae47173f6ba42b1aca3abb8c09d6d8R56-R59